apps/bsd/src/App.test.tsx

```diff
@@ -8,7 +8,7 @@ import {
   screen,
 } from '@testing-library/react'
 import { act } from 'react-dom/test-utils'
-import { electionSample } from '@votingworks/fixtures'
+import { electionSample, electionSampleDefinition } from '@votingworks/fixtures'
```

Comment:
does the original electionSample still need to be imported? We could just use electionSampleDefinition.election right?

Reply:
That would work, yeah. I was just making the minimal changes needed.